src/core/utils/index.js

```diff
@@ -336,3 +348,39 @@ export const getAddonTypeFilter = (addonType, _config = config) => {
 
   return ADDON_TYPE_THEMES_FILTER;
 };
+
+export const getExtensionTypeAction = (type = ADDON_TYPE_EXTENSION) => {
+  let action = TRACKING_TYPE_EXTENSION;
+
+  if (type === ADDON_TYPE_STATIC_THEME) {
+    action = TRACKING_TYPE_STATIC_THEME;
+  }
+
+  return action;
+};
+
+export const getAddonCategory = ({
+  type = ADDON_TYPE_EXTENSION,
+  installCategory = INSTALL_CATEGORY,
+} = {}) => {
+  let category;
+  const isTheme = ADDON_TYPE_THEMES.includes(type);
```

Comment:
Does this meet the requirements discussed at https://github.com/mozilla/addons-frontend/issues/5073#issuecomment-391068712? It seemed like we were being asked to differentiate between events for LWTs and static themes and I'm not sure we're doing that here. I may be misinterpreting the code and/or the requirements, but I think we need to be sure we're doing what's being asked.

Reply:
I *think* this is okay b/c I am updating the action property for this as this is currently the property that extension shows as 'addon' (the 1st comment mentions addressing [this](https://github.com/mozilla/addons-frontend/issues/5073#issue-325345556)). For static theme, this is where I am using 'addon:statictheme' to distinguish it from a regular extension install. 

The category piece is where I was addressing the 2nd part of the 2nd comment:

> Both stats can be added together if there's a need for an "all themes" stat, right?

Here I am using the category to show "AMO Themes [install type]" for both static theme and lightweight theme. I was thinking this could be a way to show the aggregated theme installs.

Also, it might make it clearer if you see exactly what data comes through on each type of install:
static theme:
![Alt text](https://monosnap.com/image/wlYuJv91Km6gwuUhuDhWAE4xPNRTQK.png)

lightweight theme:
![Alt text](https://monosnap.com/image/Ra3TySfRppZ4VrQWwQbmosIj7u5Pkc.png)

extension:
![Alt text](https://monosnap.com/image/D9ShhyPcRcQ855PPza9zbF7ZzHGLY6.png)

I guess there are different ways to group it and I can def move things around too if you think it might be clearer another way

Reply:
Looking at the data it does look okay to me. Thanks! I think I misunderstood how the category was being used.

Reply:
ok cool, I thought that might help "explain" more what I was thinking : )